lib/cipherstash/client.rb

```diff
@@ -73,7 +73,7 @@ def self.client_options
     #
     # @raise [CipherStash::Client::Error::InvalidProfileError] if the profile, after being overridden by environment variables and constructor options, was not valid.
     #
-    def initialize(profileName: Unspecified, logger: Unspecified, metrics: Metrics::Null.new, **opts)
+    def initialize(profileName: Unspecified, logger: Unspecified, metrics: Metrics::Null.new, rpc_class: RPC,  **opts)
```

Comment:
```suggestion
    def initialize(profileName: Unspecified, logger: Unspecified, metrics: Metrics::Null.new, rpc_class: RPC, **opts)
```

This new option needs docs.  I know (or at least *presume*) it's for internal (testing) purposes, but at the very least there needs to be a "Here Be Dragons" sign, so curious outsiders don't go getting eaten.